--- origsrc/blam-1.8.7/Makefile.am	2009-04-18 07:52:32.000000000 -0500
+++ src/blam-1.8.7/Makefile.am	2009-12-07 03:03:45.577719100 -0600
@@ -34,7 +34,7 @@ bindir=$(prefix)/bin
 man_MANS = blam.1
 
 blam: blam.in config.log
-	sed -e "s|\@prefix\@|$(prefix)|" < blam.in > blam
+	sed -e "s|\@prefix\@|$(prefix)|" < $< > $@
 
 configdir = $(prefix)/lib/blam
 config_DATA = blam.exe.config
--- origsrc/blam-1.8.7/lib/Makefile.am	2009-04-18 07:52:32.000000000 -0500
+++ src/blam-1.8.7/lib/Makefile.am	2009-12-07 03:04:21.582581900 -0600
@@ -17,10 +17,10 @@ EXTRA_DIST = $(extra_lib_DATA) \
 all: AtomFeed.dll RSSFeed.dll
 
 AtomFeed.dll: AtomFeed.cs
-	$(CSC) $(feed_FLAGS) $(af_ASSEMBLIES) $(af_SOURCES) /out:$@
+	$(CSC) $(feed_FLAGS) $(af_ASSEMBLIES) $< /out:$@
 
 RSSFeed.dll: RSSFeed.cs
-	$(CSC) $(feed_FLAGS) $(rf_SOURCES) /out:$@
+	$(CSC) $(feed_FLAGS) $< /out:$@
 
 install-data-local: AtomFeed.dll RSSFeed.dll
 	$(mkinstalldirs) $(DESTDIR)$(feed_LIBDIR)
--- origsrc/blam-1.8.7/src/Makefile.am	2009-04-18 07:55:35.000000000 -0500
+++ src/blam-1.8.7/src/Makefile.am	2009-12-07 03:05:31.361503600 -0600
@@ -3,7 +3,7 @@ CSC = $(MCS)
 FLAGS = /debug
 
 RESOURCES = \
-	    -resource:blam.glade,blam.glade                                  \
+	    -resource:$(srcdir)/blam.glade,blam.glade                              \
 	    -resource:$(top_srcdir)/icons/blam-add-news.png,blam-add-news.png      \
 	    -resource:$(top_srcdir)/icons/blam-edit-news.png,blam-edit-news.png
 
@@ -49,11 +49,11 @@ endif
 Defines.cs: Defines.cs.in ../config.log
 	sed -e "s|\@version\@|$(VERSION)|" \
 	    -e "s|\@datadir\@|$(datadir)|" \
-	    < Defines.cs.in > Defines.cs
+	    < $< > $@
 
 blamlibdir = $(prefix)/lib/blam
 blam.exe: $(BLAM_CSFILES) $(GENERATED_CSFILES) blam.glade 
-	$(CSC) $(FLAGS) -lib:$(top_srcdir)/lib $(RESOURCES) $(BLAM_LIBS) $(ASSEMBLIES) $(BLAM_CSFILES) $(GENERATED_CSFILES) -out:$@
+	$(CSC) $(FLAGS) -lib:$(top_srcdir)/lib $(RESOURCES) $(BLAM_LIBS) $(ASSEMBLIES) $(addprefix $(srcdir)/,$(BLAM_CSFILES)) $(GENERATED_CSFILES) -out:$@
 
 all: blam.exe
 
